### ISSUE TITLE
Checkstyle results [UnnecessaryParentheses]

### ISSUE THREAD

**Issue by bekuno** [cgeo/cgeo#11423]
### Describe your problem!

After the change for checkstyle we have now a lot of information reports:

~~~
Checkstyle rule violations were found. See the report at: file:///home/jenkins/slave/workspace/cgeo%20pull%20request/main/build/reports/checkstyle/checkstyle.html
Checkstyle files with violations: 38
Checkstyle violations by severity: [info:267]
~~~

This should be fixed.
I think that it is now ok only to add `@SuppressWarnings("checkstyle:UnnecessaryParentheses")` to the methods.

### How to reproduce?

see eg. https://ci.cgeo.org/job/cgeo%20pull%20request/6672/console

gradlew checkstyle

### Reproducible

Yes

### c:geo Version

release after PR #11399

**Comment by fm-sys** [comment]
> I think that it is now ok only to add `@SuppressWarnings("checkstyle:UnnecessaryParentheses")` to the methods.

Haven't we agreed on rather disabling the check completely instead of annotating all methods?

**Comment by bekuno** [comment]
No.
And no reaction to https://github.com/cgeo/cgeo/pull/11399#issuecomment-893862793 so far.

**Comment by eddiemuc** [comment]
I looked into documentation of checkstyle and found out you can exclude certain expressions from "unnecessary parenthesis" check. I did this for "logical and" expression (e.g. a parenthesis around such an expression will no longer be flagged as unnecessary), see PR #11425. I propose this as a fix for #11261

**Comment by bekuno** [comment]
close here in favor of issue #11261
